ui/src/main/resources/edu/wpi/grip/ui/templates/cpp/operations/Normalize.vm

```diff
@@ -0,0 +1,19 @@
+	enum NormalizeType {
+	   INF = NORM_INF,
+	   L1 = NORM_L1,
+	   L2 = NORM_L2,
+	   MINMAX = NORM_MINMAX
+	   };
```

Comment:
Why are you generating an enum for this? Just map the enum values from GRIP to the OpenCV values (so an operation using `MINMAX` should call the method with `NORM_MINMAX`). This goes for every other enum use; they're all just wrapping OpenCV constants